app/model.py

```diff
@@ -0,0 +1,45 @@
+from pydantic import BaseModel, EmailStr, validator, Field
+
+
+class User(BaseModel):
+    username: EmailStr
+    password: str
+    user_type: str
+    class Config:
+        schema_extra ={
+            "user_credentials":{
+                "username":"alekhya@gmail.com",
+                "password":"ale",
+                "user_type":"admin"
+            }
+        }
+
+
+class Ngo(BaseModel):
+    id:int 
+    ngoName: str = Field(default=None)
+    contactPersonName: str
+    contactPersonEmail: EmailStr
+    contactPersonPhone: int = Field(default=None)
+    contactPersonPassword: str
+    location: str
+    pincode: int
+    class Config:
+        schema_extra ={
```

Comment:
Is it fine to pass the value as a hardcoded ?

Reply:
i have done this as a basic idea only...it is not connecting with database....first i explored this on auth part....